runners/spark/3/src/main/java/org/apache/beam/runners/spark/structuredstreaming/translation/SparkSessionFactory.java

```diff
@@ -29,6 +29,7 @@
 import java.util.HashMap;
 import javax.annotation.Nullable;
 import org.apache.beam.repackaged.core.org.apache.commons.lang3.ArrayUtils;
+import org.apache.beam.runners.core.construction.SerializablePipelineOptions;
```

Comment:
ugh

Reply:
This was not intended, probably some wrong targeted replacement. Will fix that in different PR.

Reply:
Ah, sorry, I forgot. The discusson with @robertwb was that we will remove SerializablePipelineOptions from sdks-java-core completely. That is why the packages changed in all affected runners. Actually, I think that is correct, because of the potential serializability issue. The SerializablePipelineOptions is intended for runners, so runners-core-java might be actually the correct place to put it.

Reply:
Yes. 

SerializablePipelineOptions (per its name) is presumably intended for serialization, so it would be backwards incompatible to move it. Fortunately it's only referenced by runners, so probably makes sense as part of a runner support package. I agree further cleanup could be done here, but that would require resolving the shading/jackson issues too.